### ISSUE TITLE
Missing Objects

### ISSUE THREAD

**Issue by PatWie** [svs14/Orchestra.jl#5]
I tried to use this modul. Unfortunately, therse is no hint about compatible versions of Julia.

```
julia> import Orchestra
ERROR: BaggingClassifier not defined
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in reload_path at loading.jl:152
 in _require at loading.jl:67
 in require at loading.jl:51
while loading /home/patwie/.julia/v0.3/Orchestra/src/python/scikit_learn.jl, in expression starting on line 21
while loading /home/patwie/.julia/v0.3/Orchestra/src/transformers.jl, in expression starting on line 47
while loading /home/patwie/.julia/v0.3/Orchestra/src/Orchestra.jl, in expression starting on line 8
```

I (pip) installed all python stuff in my home dir.

**Comment by svs14** [comment]
Thanks for bringing this up.
The earliest I can look at this issue in full is in 2 days.

That's correct, I'll patch up the REQUIRE file to fix at this at Julia 0.3 until I've tested 0.4.

I can't verify this yet, but my guess is that you are running a version that is not 0.15 for scikit-learn, of course that means Orchestra.jl should be updated to the latest release (0.17?) on pip so I'll have look at this as soon as I can.

**Comment by PatWie** [comment]
The only working combination for me is julia0.3 (0.3.9) and scikit-learn 0.15. Removing scikit-learn and

```
pip install scikit-learn==0.15
```
